docs/data/data-grid/api-object/api-object.md

```diff
@@ -60,6 +60,37 @@ If you try to use it in the first render of the component, it will crash because
 
 ## Common use cases
 
+### Access the disabled column features
```

Comment:
I we could state more clearly that the flags control the UI just like `[disabled]` controls an `input[type=text]`. Not sure if it's better on each flag's section or here.

Reply:
I wanted to add it here along with each section and possibly an upcoming FAQ section to have better discoverability.